.tool-versions.sample

```diff
@@ -1,3 +1,3 @@
-ruby 2.6.10
+ruby 2.7.6
 nodejs 14.21.3
 python 2.7.18
```

Comment:
Do we use python?

Reply:
I think this is something for macOS...

Reply:
Yeah, no sensible people should be concerned...

Reply:
It could be commented or removed if it's a mac-only thing